librz/debug/p/native/linux_arm64.c

```diff
@@ -655,15 +655,6 @@ static int rz_debug_native_drx(RzDebug *dbg, int n, ut64 addr, int sz, int rwx,
 #define PTRACE_SETHBPREGS 30
 #endif
 
-static bool ll_arm32_hwbp_set(pid_t pid, ut64 addr, int size, int wp, int type) {
```

Comment:
don't delete functions that are used.

Reply:
if needed i can test this on my arm64 machine

Reply:
Yeah it will be very helpful if you can tell me the error in arm64
For now, I've figured out how to resolve the other two and as for arm64, the function being there or not doesn't make any difference because it isn't being called infact it was not supposed to be there in the first place, I think I forgot to remove it while refactoring code for the previous PR.
But I don't exactly know the issue yet for arm64.

Reply:
check the ` Travis CI` output

Reply:
seems that `arm64_hwbp_add` and `arm64_hwbp_del` are not defined. i guess it's missing their declaration

Reply:
arm64 build also gives this error : `/librz/debug/p/native/linux_arm64.c:784: undefined reference to `linux_generate_corefile'`
Function `linux_generate_corefile` is always defined when `__arm64__`,`__i386__`,`__arm__` or `__x86_64__` are present.
And `linux_arm64.c` is compiled when `__arm64__` and `__aarch64__` macros are present.
So for this build, `__arm64__` macro is absent but `__aarch64__` is present unlike other android and linux arm64 builds where both macros are present.

Reply:
Tonight/tomorrow I will pull your changes and see